templates/galaxy/config/file_sources_conf.yml.j2

```diff
@@ -207,3 +207,9 @@
   bucket: mf-nwp-models
   anon: true
 
+- type: gitlabfs
+  id: gitlab_fs
+  label: DataPLANT ARC Gitlab
+  doc: Explore ARCs in the DataPLANT Gitlab repository
+  token: ${user.preferences['gitlabfs|read_access_token']}
```

Comment:
read_access_token looks not defined? Should this be just `token`?

Reply:
Absolutely! Thanks for the catch!